### DISCUSSION TITLE
[Backend] Spring Boot + GraalVM + Spring Data JPA + Spring Security

### DISCUSSION THREAD

**Discussion by szymonprz** [realworld-apps/realworld#1316] [WIP Implementations]
I have just published a new implementation of a real world application where we're using graalvm to build a native image with spring boot

Here's the link to repo: https://github.com/softwaremill/realworld-spring-boot-native

**Comment by szymonprz** [comment]
Please check it out @geromegrignon 🎉

**Comment by alejandrolorefice** [comment]
This is great Szymon! I'll give it a try later :raised_hands:
